### DISCUSSION TITLE
Is there a data sheet for the display?

### DISCUSSION THREAD

**Discussion by robertbastian** [Seeed-Solution/SenseCAP_Indicator_ESP32#41] [Q&A] [answered]
I've gotten most of the peripherals working in Rust, however I'm struggling with the display. Which protocol should I use?

**Comment by Love4yzp** [comment]
Alright, I'll check with the engineer and might be able to give you an answer after February 17th.

**Reply by Love4yzp** [reply, marked as answer]
RGB: 480*480
Touch IC: FT6336U ( I2C)
Display IC: ST7701S

You can find here: [ST7701S - PDF](https://community.nxp.com/pwmxy87654/attachments/pwmxy87654/imx-processors/134161/3/ST7701S_SPEC_Preliminary%20V0.1.pdf)

**Reply by robertbastian** [reply]
Thank you
